src/main/java/com/lineate/testyourlexicon/dto/SingleTranslationDto.java

```diff
@@ -0,0 +1,14 @@
+package com.lineate.testyourlexicon.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Builder
+@Data
+public class SingleTranslationDto {
```

Comment:
Let's declare this class as a model since it is describing a business logic entity. Probably the name can be just `Translation`.